utility.py

```diff
@@ -0,0 +1,68 @@
+from concurrent import futures
+from azure.storage.blob import BlockBlobService, PublicAccess
+import os, uuid,sys
+import io
+import grpc
+import time
+import os.path
+import config
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+
+CHUNK_SIZE = 1024 * 1024
+
+def download_chunk(file):
+     with open(file, 'rb')as f:
+         while True:
+             chunk = f.read(CHUNK_SIZE)
+             if len(chunk) == 0:
+                 return
+             yield hwsc_file_transaction_svc_pb2.chunk(buffer=chunk)
+
+def get_file_type(fileName):
+    fileType = ''
+    imageRegex = set('.jpg .jpeg . png .bmp . tif .gif .tiff'.split())
+    audioRegex = set('.wav .wma . ogg . m4a .mp3'.split())
+    videoRegex = set('.flv .wmv .mov .avi .mp4'.split())
+    fileRegex = set('.doc .txt .mat'.split())
+
+    _, extension = os.path.splitext(fileName)
+    if extension in imageRegex:
+        fileType = 'images'
+    elif extension in audioRegex:
+        fileType = 'audios'
+    elif extension in videoRegex:
+        fileType = 'videos'
+    elif extension in fileRegex:
+        fileType = 'files'
+    return fileType
```

Comment:
I don't think you are using REGEX here. This is also a very inefficient linear or (O(4N) to be exact) solution. Consider reading this https://stackoverflow.com/questions/185378/regular-expression-to-match-start-of-filename-and-filename-extension